docs/src/content/hardhat-runner/docs/guides/command-line-completion.md

```diff
@@ -1,17 +1,21 @@
 # Command-line completion
 
-`hardhat-shorthand` is an NPM package that installs a globally accessible binary called `hh` that runs the project's locally installed `hardhat` and supports shell autocompletion for tasks.
+Hardhat has a companion NPM package that acts as a shorthand for `npx hardhat`, and at the same time, it enables command-line completions in your terminal.
+
+This package, `hardhat-shorthand`, installs a globally accessible binary called `hh` that runs your locally installed `hardhat`.
 
 ## Installation
 
 To use the Hardhat shorthand you need to install it **globally**:
 
 ```
-npm i -g hardhat-shorthand
+npm install --global hardhat-shorthand
 ```
```

Comment:
Should we use tabs here? It doesn't feel as necessary as in other places, but maybe?

Reply:
I was a bit confused about this. As any of them would work. Maybe it's simpler to recommend a single option. Less to debug if people follow it and have problems.

Reply:
Nah, let's be consistent.

Reply:
Nop, I changed my mind. This works differently in yarn 1 and 2. Let's stick to npm.